components/atom/input/README.md

```diff
@@ -0,0 +1,114 @@
+# AtomInput
+
+> Description
+
+<!-- ![](./assets/preview.png) -->
+
+## Installation
+
+```sh
+$ npm install @schibstedspain/sui-atom-input --save
+```
+
+## Usage
+
+### Basic usage
+```js
+import MaskInput from '@schibstedspain/sui-atom-input/lib/inputs/MaskInput'
+import AtomInput from '@schibstedspain/sui-atom-input'
+
+// render the same
+return (
+  <div>
+    <MaskInput label='postal code' name='date' mask={postalCodeMask}/>
```

Comment:
What about this ?

```js
import {Mask} from '@schibstedspain/sui-atom-input'
```

the option of `/lib/inputs/MaskInput` is only for optimization purpose I guess ?

Anyways I would only show one way to do it.

Reply:
I have no preference about it, maybe someone else want to give his opinion about this

Yes, it's "optimization"